lib/media/media_source_engine.js

```diff
@@ -349,10 +357,8 @@ shaka.media.MediaSourceEngine = class {
               shaka.media.Transmuxer.convertTsCodecs(contentType, mimeType);
         }
         const sourceBuffer = this.mediaSource_.addSourceBuffer(mimeType);
-        if (sequenceMode) {
-          sourceBuffer.mode =
-              shaka.media.MediaSourceEngine.SourceBufferMode_.SEQUENCE;
-        }
+        sourceBuffer.mode =
```

Comment:
Do we need to manually set the source buffer mode to SEGMENTS?
According to MDN:

> If you try to set the `mode` property value to `segments` when the initial value is `sequence`, an exception will be thrown.

So it doesn't seem like there's a situation where this would be both necessary and possible.

Reply:
No, I suppose not.  It was just created, and that's the default.

Reply:
Dropped.